plugins/inputs/tail/multiline_test.go

```diff
@@ -234,3 +238,110 @@ func TestMultilineWhat(t *testing.T) {
 	require.Error(t, w7.UnmarshalTOML([]byte(`nope`)))
 	require.Equal(t, MultilineMatchWhichLine(-1), w7)
 }
+
+func TestMultiLineQuoted(t *testing.T) {
+	tests := []struct {
+		name      string
+		quotation string
+		quote     string
+		filename  string
+	}{
+		{
+			name:      "single-quotes",
+			quotation: "single-quotes",
+			quote:     `'`,
+			filename:  "multiline_quoted_single.csv",
+		},
+		{
+			name:      "double-quotes",
+			quotation: "double-quotes",
+			quote:     `"`,
+			filename:  "multiline_quoted_double.csv",
+		},
+		{
+			name:      "backticks",
+			quotation: "backticks",
+			quote:     "`",
+			filename:  "multiline_quoted_backticks.csv",
+		},
+	}
```

Comment:
can you add a test case with a "missing" quote, both inside and outside the data. Want to ensure we error correctly